src/Jobs/Recruit.Vacancies.Jobs/DomainEvents/Handlers/Vacancy/VacancyClonedDomainEventHandler.cs

```diff
@@ -6,12 +6,12 @@
 
 namespace Esfa.Recruit.Vacancies.Jobs.DomainEvents.Handlers.Vacancy
 {
-    public class VacancyCreatedHandler : DomainEventHandler, IDomainEventHandler<VacancyCreatedEvent>
+    public class VacancyClonedDomainEventHandler : DomainEventHandler, IDomainEventHandler<VacancyClonedEvent>
```

Comment:
Do we not need a VacancyCreatedDomainEventHandler?

Is the @event also incorrect?

Reply:
So we still have a `VacancyCreated` event which is used by the `UpdateEmployerDashboardOnChange` errmmm in proc `EventHandler` 😕  but there is nothing out of process that we want to do on this event which is why we don't need it for a purpose. The event is still being stored in the domain events queue though just because of the nature of the event.

Reply:
Why re-publish it at all? It blows up at the moment I think due to there being no handler registered for the event type.

This type is incorrect isn't it?

            var @event = DeserializeEvent<VacancyCreatedEvent>(eventPayload);

Reply:
👍 Didn't notice that 🔫